python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -51,26 +81,44 @@ python paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
 ```
 The converted data binary file is saved by default in `~/.cache/paddle/dataset/int8/download/int8_full_val.bin`
  * ### Prepare model
-You can run the following commands to download ResNet50 model.
+You can run the following commands to download ResNet50 model. Since for performance optimization, we should be aware of the graph optimization during the QAT training phase, so we have the different fake QAT models between QAT MKL-DNN 1.0 and QAT MKL-DNN 2.0.
```

Comment:
`The exemplary code snippet provided below downloads a ResNet50 QAT model. The reason for having two different versions of the same model originates from having a two different QAT training strategies: One for an non-optimized and second for an optimized graph transform which correspond to QAT 1.0 and QAT 2.0 respectively.`

Reply:
Done